env

```diff
@@ -1,10 +1,11 @@
-goversion="1.18.1"
-DAEMON=simd
-DENOM=stake
+goversion="1.19"
+DAEMON=umeed
+DENOM=uumee
 CHAINID=test
-DAEMON_HOME=${HOME}/.simd
-GH_URL=https://github.com/cosmos/cosmos-sdk
-CHAIN_VERSION='v0.45.4'
+DAEMON_HOME=${HOME}/.qa-integration/umeed
+GH_URL=https://github.com/umee-network/umee
+CHAIN_VERSION='main'
+IS_PUBLIC=false
 UPGRADE_NAME='v045-to-v046'
 UPGRADE_VERSION='v0.46.0-rc2'
 NUM_TXS=50
```

Comment:
maybe we should create other file: `env-umee` instead of modifying this one?

Reply:
can be, but then at `env-check.sh` should we source the env-umee ?

Reply:
done at https://github.com/umee-network/qa-integration/pull/2/commits/a76f1dcf2799f30bd2676a118613c9fdd29c2440